docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -998,8 +998,7 @@ connector using a {doc}`WITH </sql/create-table-as>` clause.
     for row level deletes.
 * - `max_commit_retry`
   - Number of times to retry a commit before failing. Defaults to the value of 
-    the `iceberg.max-commit-retry` catalog configuration property, which 
-    defaults to `4`.
+    the `iceberg.max-commit-retry` catalog configuration property.
```

Comment:
Document should still mention the iceberg default

Reply:
I removed it because I was worried about the upstream change. I will revert anyway.